runtime/vm/jvminit.c

```diff
@@ -6366,21 +6366,25 @@ protectedInitializeJavaVM(J9PortLibrary* portLibrary, void * userData)
 			 *
 			 * OpenJ9 stores both of these values as UDATA, so they will both be printed
 			 * as "size_t" type.
+			 *
+			 * NOTE that Hotspot produces this output on STDOUT, which means this
```

Comment:
```suggestion
			 * NOTE that Hotspot produces this output on STDOUT, and applications expect to parse it there,
			 * which means this ....
```

Reply:
In my mind, the more important part is that apps expect it to be output to `stdout` for parsing.  Does Hotspot produce all their output to `stdout`?  That wouldn't be sufficient to move everything from `stderr` -> `stdout` :)